### PR TITLE
Fix light eater not being able to break mech lights or wisps

### PR DESCRIPTION
**Pull request by ItsTisme**
## About The Pull Request

The wisp had no lighteater_act implemented. Now it also has a nice signal attached, mechs caused runtimes because it called proc for the wrong lighting method

## Why It's Good For The Game

fixes a runtime and wisp being disablable is likely intended

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

not sure what to add here

</details>

## Changelog
:cl:
tweak: light eater works on wisps
fix: Light eater works on mechs
/:cl:

### CODE REVIEW

**Comment by Autisem** [comment]
actuely just figurd of an edge case, leme fix that

File: code/game/atoms.dm [hunk 1]
```diff
@@ -802,7 +802,7 @@
   * Called when lighteater is called on this.
   */
 /atom/proc/lighteater_act(obj/item/light_eater/light_eater)
-	return
+	SEND_SIGNAL(src,COMSIG_ATOM_LIGHTEATER_ACT)
```
Comment (Kapu1178) [inline code/game/atoms.dm:807]:
```suggestion
	SHOULD_CALL_PARENT(TRUE)
	SEND_SIGNAL(src,COMSIG_ATOM_LIGHTEATER_ACT)
```

Comment (ItsTisme) [reply code/game/atoms.dm:807]: just wondering what this would do,

**Comment by ItsTisme** [comment]
Right so reverting the `SHOULD_CALL_PARENT(TRUE)` because the borg camera explicity overwrites it to NOT call the parent, refeer to PR #6201

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -345,6 +359,23 @@
...
+		src.forceMove(home)
+		time = world.time + cooldown
+		home.icon_state = "lantern-blue"
+		set_light_on(FALSE)
+	else
+		stop_orbit()
+		qdel(src)
+
+
+
```
Comment (Archanial) [inline code/modules/mining/lavaland/necropolis_chests.dm]:
```suggestion
```

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -289,20 +289,29 @@
 		return
 
 	if(wisp.loc == src)
-		to_chat(user, "<span class='notice'>You release the wisp. It begins to bob around your head.</span>")
-		icon_state = "lantern"
-		wisp.orbit(user, 20)
-		SSblackbox.record_feedback("tally", "wisp_lantern", 1, "Freed")
+		if(wisp.time < world.time)
```
Comment (Archanial) [inline code/modules/mining/lavaland/necropolis_chests.dm]: use our cooldown defines

**Review by Archanial — CHANGES_REQUESTED** [review]
I agree with Kapu here, every lighteater act should send the signal.

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 3 above)
Comment (ItsTisme) [reply code/modules/mining/lavaland/necropolis_chests.dm]: Where are these defined? or just a term that I can quickly search in VSCode

**Review by Archanial — APPROVED** [review]

**Pull request merged**